docs/src/main/sphinx/release/release-353.md

```diff
@@ -0,0 +1,87 @@
+# Release 353 (XXXXXXXX March 2021)
+
+## General
+
+* Fix aliasing of columns in common table expressions. ({issue}`6839`)
+* Fix potential incorrect query results for queries containing multiple `<` predicates. ({issue}`6896`)
+* Always show `SECURITY` clause in `SHOW CREATE VIEW`. ({issue}`6720`)
+* Fix reporting column references for aliased tables. ({issue}`6972`)
```

Comment:
where can the user see it? 

cc @sopel39 @kasiafi

Reply:
From the original issue:
https://github.com/trinodb/trino/issues/6960#issue-812050046

Reply:
Maybe `Fix reporting of column references to event listener for aliased tables`